src/Liquidator.sol

```diff
@@ -207,8 +168,8 @@ contract Liquidator is Owned, ILiquidator {
      * @param minPriceMultiplier_ The new minimum price multiplier, with 2 decimals precision.
```

Comment:
```suggestion
     * @param minPriceMultiplier_ The new minimum price multiplier, with 4 decimals precision.
```
Maybe do a ctrl-f replace on 2 decimals in case we missed other comments.